js/mviewer.js

```diff
@@ -2745,6 +2852,10 @@ mviewer = (function () {
 
         getSourceOverlay: function () { return _sourceOverlay; },
 
+        getSourceSelectOverlay: function () { return _sourceSelectOverlay; },
+
+        getSourceSubSelectOverlay: function () { return _sourceSubSelectOverlay; },
```

Comment:
You really need to expose sources ?

Reply:
No, thanks for again for your attention!